test/controllers/api/v1/tasks_controller_test.rb

```diff
@@ -0,0 +1,55 @@
+require 'test_helper'
+
+class Api::V1::TasksControllerTest < ActionController::TestCase
+  test 'should get show' do
+    author = create(:user)
```

Comment:
this method accrues in almost every test. lets move it to `setup` block 

tests must be as light as possible